File: botindex.py
import os
import discord
from dotenv import load_dotenv
from discord.ext import commands

load_dotenv()
TOKEN = os.getenv('DISCORD_TOKEN')

client = discord.Client()
client = commands.Bot(command_prefix = '!')
@client.command()
async def load(ctx, extension):
    client.load_extension(f'cogs.{extension}')

@client.command()
async def unload(ctx, extension):
    client.unload_extension(f'cogs.{extension}')

for filename in os.listdir('./cogs'):
    if filename.endswith('.py'):
        client.load_extension(f'cogs.{filename[:-3]}')

@client.event
async def on_ready():
    print(f'{client.user} has connected to Discord!')
# print(TOKEN)
client.run(TOKEN)

File: botindex.js
const Discord = require('discord.js');
const { prefix, token } = require('./config.json');
const bot = new Discord.Client();
const fs = require('fs');
const startTime = Date.now();
bot.commands = new Discord.Collection();

function titleCase(str) {
	var splitStr = str.toLowerCase().split(' ');
	return splitStr; 
 };
 
fs.readdir('./commands/', (err, files) => {
	if(err) console.error(err);

	var jsfiles = files.filter(f => f.split('.').pop() === 'js');
	if (jsfiles.length <= 0){
		return console.log('No commands found...')
	} else {
		console.log(jsfiles.length + ' commands found.')
	}

	jsfiles.forEach((f, i) => {
		var cmds = require(`./commands/${f}`);
		console.log(`Command ${f} loading...`)
		bot.commands.set(cmds.configure.command, cmds);
	})
})

bot.on('ready', () => {
	console.log('Ready!\nUse this link to add the bot to your disord server! \nhttps://discord.com/api/oauth2/authorize?client_id=713815087449767997&permissions=67648&scope=bot');
	console.log(`Bot has started with ${bot.users.cache.size} users in ${bot.channels.cache.size} channels on ${bot.guilds.cache.size} servers.`); 
	bot.user.setStatus("online");
	bot.user.setActivity(`!help | ${bot.users.cache.size} Users!`);
	//${bot.guilds.size} Servers |
});

bot.on('message', message => {
	if (!message.content.startsWith(prefix)) return;
	var cont = message.content.slice(prefix.length).split(' ');
	var cscont = titleCase(cont.join(' '));
	var args = cscont.slice(1);
	var sender = message.author;
	var cmd = bot.commands.get(cscont[0])
	if (cmd) cmd.run(bot,message, args);
});

bot.login(token);